src/main/scala/org/ergoplatform/mining/CandidateGenerator.scala

```diff
@@ -723,6 +721,9 @@ object CandidateGenerator extends ScorexLogging {
                     val boxesToSpend = feeTx.inputs.flatMap(i =>
                       newBoxes.find(b => java.util.Arrays.equals(b.id, i.boxId))
                     )
+                    implicit val verifier: ErgoInterpreter = ErgoInterpreter(
+                      us.stateContext.currentParameters
+                    )
```

Comment:
I remember `implicit` was removed and the verifier was moved out of the loop. And also resetContext was used. Something changed?

Reply:
I was also thinking why it cannot be outside of the loop, initialized only once

Reply:
That is done in another PR https://github.com/ergoplatform/ergo/pull/1522